draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -630,7 +630,7 @@
    <t>
    The CI/T interface builds on top of HTTP, so dCDNs may make use of any HTTP feature when implementing the CI/T interface. 
    For example, dCDN SHOULD make use of HTTP's caching mechanisms to indicate that a requested resource representation 
-   has not been modified, reducing uCDN's processing needed to determine whether the status of a trigger has changed.
+   has not been modified, reducing uCDN's processing is needed to determine whether the status of a trigger has changed.
```

Comment:
This change is grammatically incorrect.  Either the "is" should be removed, or the latter statement should be changed to its own sentence.

Reply:
Rewrote as "For example, dCDN SHOULD make use of HTTP's caching mechanisms to indicate that a requested resource representation has not been modified, to reduce uCDN's overhead of polling of the trigger status."